.github/workflows/pmm-ui-tests-upgrade.yml

```diff
@@ -138,111 +145,124 @@ jobs:
           target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
           sha: ${{ env.SHA }}
 
-      - name: Install NodeJS v16
-        uses: percona-platform/setup-node@v2
-        with:
-          node-version: 16.14.1
-
-      - name: Checkout UI tests
-        uses: percona-platform/checkout@v2
+      - name: 'Checkout UI tests: "${{ inputs.pmm_qa_branch }}"'
+        uses: actions/checkout@v3
         with:
-          ref: ${{ env.UI_TESTS_BRANCH }}
+          ref: ${{ inputs.pmm_qa_branch }}
```

Comment:
QA branch for UI tests?

Reply:
Not sure I got the question. Could you please be more specific on what is your concern in using `inputs.value` instead of creating `env.Variable` ?

Reply:
fixed